packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.base.tsx

```diff
@@ -21,12 +21,16 @@ export class ColorRectangleBase extends BaseComponent<IColorRectangleProps, ICol
     minSize: 220
   };
 
+  private _eventsGroup: EventGroup;
   private _root = React.createRef<HTMLDivElement>();
 
   constructor(props: IColorRectangleProps) {
     super(props);
 
-    this._warnDeprecations({
+    initializeComponentRef(this);
```

Comment:
All the stuff in this file is only needed if you're removing the BaseComponent inheritance.

Reply:
Removed BaseComponent